articles/virtual-machines/windows/multiple-nics.md

```diff
@@ -183,6 +183,13 @@ $vm.NetworkProfile.NetworkInterfaces
 Remove-AzureRmNetworkInterface -Name "myNic3" -ResourceGroupName "myResourceGroup" | `
     Update-AzureRmVm -ResourceGroupName "myResourceGroup"
 ```
+^^^^ THIS IS THE WRONG CMDLET AND THE WRONG TYPE OF DATA FOR "-Name" ANYWAY  ^^^^

```

Comment:
Thanks for the feedback, sorry it's incorrect. If you can remove these two lines (186 + 187), we can merge the updates.